extensions/ipynb/package.json

```diff
@@ -70,16 +80,19 @@
 		}
 	},
 	"scripts": {
-		"compile": "npx gulp compile-extension:ipynb",
-		"watch": "npx gulp watch-extension:ipynb"
+		"compile": "npx gulp compile-extension:ipynb && npm run build-notebook",
+		"watch": "npx gulp watch-extension:ipynb",
+		"build-notebook": "node ./esbuild"
 	},
 	"dependencies": {
 		"@enonic/fnv-plus": "^1.3.0",
 		"detect-indent": "^6.0.0",
+    "markdown-it": "^12.3.2",
```

Comment:
You likely don't need this dependency as you are only using the types from `markdown-it`